var/spack/repos/builtin/packages/py-tensorpack/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyTensorpack(PythonPackage):
+    """Tensorpack is a neural network training interface based
+    on TensorFlow."""
+
+    homepage = "https://github.com/tensorpack/tensorpack"
+    pypi = "tensorpack/tensorpack-0.10.1.tar.gz"
+
+    version('0.10.1', sha256='ae6af59794459de910725d268061f0c86d78f01948f9fd5d7b11dd9770ad71ef')
```

Comment:
Would you be willing to add yourself as the Spack package manager for this package?

Reply:
I don't know if I'm the best person to list as maintainer for any of the packages I push up. My interaction with actual software that gets installed is VERY minimal. Once our users are happy and the package gets accepted here, I move on to the next package and forget most of the specifics about the last.

Reply:
@aweits may be interested in being a maintainer for this

Reply:
@qwertos you can put me as maintainer...